scheduling/task_run_list.go

```diff
@@ -0,0 +1,33 @@
+package scheduling
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+type TaskRunSimple struct {
+	RunID       TaskRunID `json:"runID"`
+	DateCreated time.Time `json:"dateCreated"`
```

Comment:
nit: CreatedAt

Reply:
we're already using `dateCreated` throughout